src/BizHawk.Client.Common/RomLoader.cs

```diff
@@ -990,6 +990,8 @@ private static class RomFileExtensions
 
 			public static readonly IReadOnlyCollection<string> Doom = new[] { "wad" };
 
+			public static readonly IReadOnlyCollection<string> DOS = new[] { "ima", "img", "conf", "dosbox-iso", "dosbox-cue", "dosbox-bin", "dosbox-mdf", "dosbox-chf", "xdf", "dmf", "fdd", "fdi", "nfd", "d88" };

```

Comment:
Similar to https://github.com/TASEmulators/BizHawk/pull/4264#discussion_r1992915231, these "dosbox-*" CD extensions are unacceptable. The core must use BizHawk's CD interface, it is not allowed to handle CDs themselves. This is a blocker for merge.

Reply:
Fixed

Reply:
They're still mentioned in `FileID.cs`